app/backend/approaches/chatreadretrieveread.py

```diff
@@ -99,16 +99,21 @@ def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]) -> A
         else:
             prompt = prompt_override.format(sources=content, chat_history=self.get_chat_history_as_text(history), follow_up_questions_prompt=follow_up_questions_prompt)
 
+        messages = self.get_messages_from_prompt(prompt)
+
         # STEP 3: Generate a contextual and content specific answer using the search results and chat history
-        completion = openai.Completion.create(
-            engine=self.chatgpt_deployment, 
-            prompt=prompt, 
+        chatCompletion = openai.ChatCompletion.create(
+            deployment_id=self.chatgpt_deployment,
+            model="gpt-3.5-turbo",
+            messages=messages, 
             temperature=overrides.get("temperature") or 0.7, 
             max_tokens=1024, 
             n=1, 
             stop=["<|im_end|>", "<|im_start|>"])
```

Comment:
Does "<|im_start|>" still exist in the messages? Or just "<|im_end|>"? Just checking that the stop parameter is still correct.

Reply:
We can remove stop keywords since new conversation templates do not expect it.